### ISSUE TITLE
(personal preference?) there should be space between visibility modifier and ctor params

### ISSUE THREAD

**Issue by sjrd** [olafurpg/scalafmt#106]
Original:
```scala
class WebGLFramebuffer private () extends js.Object
```

Scalafmt:
```scala
class WebGLFramebuffer private() extends js.Object
```

I would like the input to look like this:
```scala
class WebGLFramebuffer private () extends js.Object
```

Using
* 0.1.1
* default

**Comment by olafurpg** [comment]
I'll keep this non-configurable and see if someone disapproves. The fix is basically to space if there is a modifier token before opening `(`.

**Comment by olafurpg** [comment]
Fixed in #118
